tag-historian/nitaghistorian.yml

```
@@ -147,6 +147,9 @@ definitions:
         example:
           - system1.tag1
           - system2.tag1
+      workspace:
+        description: The workspace Id for querying decimated tag values. If this is not specified, the tag values from the default workspace will be returned.
+        example: 0c80cf49-54e9-4e92-b117-3bfa574caa84
```

These all need to include the `type: string` since the default is object.

```suggestion
      workspace:
        description: The workspace Id for querying decimated tag values. If this is not specified, the tag values from the default workspace will be returned.
        type: string
        example: 0c80cf49-54e9-4e92-b117-3bfa574caa84
```

---

good catch